### PR TITLE
docs: add PD Scheduler config file

### PR DESCRIPTION
**Pull request by lilin90**
Signed-off-by: lilin90 <lilin@pingcap.com>

## What have you changed? (mandatory)

Add the document: PD Scheduler Configuration

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

Relevant.

## Does this PR affect tidb-ansible update? (mandatory)

No.

### CODE REVIEW

**Comment by lilin90** [comment]
@rleungx @CaitinChen PTAL

File: docs/pd-scheduler.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+`disable-remove-extra-replica` is the option to prevent the replica checker from removing extra replicas.
+
+### `disable-location-replacement`
+
+`disable-location-replacement` is the option to prevent the replica checker from moving the replica to a better location.
+
+## Customization
+
+The default schedulers includes `balance-leader`, `balance-region` and `hot-region`. In addition, you can also customize the schedulers. For each scheduler, the configuration has three variables: `type`, `args` and `disable`.
```
Comment (CaitinChen) [inline docs/pd-scheduler.md]: includes -> include

File: docs/pd-scheduler.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+The basic workflow of PD Scheduler is as follows. First, the scheduler is triggered according to `minAdjacentSchedulerInterval` defined in `ScheduleController`. Then it tries to select the source store and the target store, create the corresponding operators and send a message to TiKV to do some operations.
+
+## Usage description
+
+This section describes the usage of PD Scheduler parameters.
+
+### `max-merge-region-keys && max-merge-region-size`
+
+If the Region size is smaller than `max-merge-region-size` and the number of rows in the Region is smaller than `max-merge-region-keys` at the same time, the Region will try to merge with adjacent Regions. The default value of both the two parameters is 0. Currently, `merge` is not enabled by default.
```
Comment (rleungx) [inline docs/pd-scheduler.md]: s/rows/keys

**Comment by lilin90** [comment]
@breeswish @CaitinChen PTAL

**Comment by CaitinChen** [comment]
@lilin90  Thanks    @breeswish  PTAL

**Comment by Connor1996** [comment]
why delete `docs/tikv-storage-configuration.md` file

**Comment by lilin90** [comment]
@Connor1996 Because it's a duplicated copy of `docs/tikv-storage-config.md`.

**Pull request merged**